applications/plugins/CGALPlugin/CMakeLists.txt

```diff
@@ -44,6 +47,12 @@ find_package(MPFR REQUIRED)
 find_package(GMP REQUIRED)
 find_package(Boost QUIET REQUIRED COMPONENTS system thread)
 
+message(STATUS "CGAL VERSION ${CGAL_MAJOR_VERSION}.${CGAL_MINOR_VERSION}")
+set(CGAL_VERSION ${CGAL_MAJOR_VERSION}.${CGAL_MINOR_VERSION})
+if (CGAL_VERSION VERSION_GREATER_EQUAL 4.10)
```

Comment:
There is a problem with this line on CI: https://ci.inria.fr/sofa-ci-dev/job/sofa-framework/job/PR-783/CI_CONFIG=ubuntu_gcc-5.4,CI_PLUGINS=options,CI_TYPE=release/1/console

`if ("${CGAL_VERSION}" VERSION_GREATER_EQUAL "4.10")` ?

Reply:
Actually it's a CMake version problem. To stay compliant with 3.1, you can't use VERSION_GREATER_EQUAL, you have to use both VERSION_GREATER and VERSION_EQUAL.

Reply:
:(
V3.1 is so 2016...